packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -77,7 +86,8 @@ const tabLightThemeStyles = css({
   '--workspace-tab-selected-background-color': palette.white,
   '--workspace-tab-border-color': palette.gray.light2,
   '--workspace-tab-color': palette.gray.base,
-  '--workspace-tab-selected-color': palette.green.dark2,
+  '--workspace-tab-selected-color': palette.black,
```

Comment:
Changed the selected text colour to match the design. Although now it is out of sync with the active text colour in the sidebar, so maybe I should revert it for now.

Reply:
Reverted some of it.